web/channels/user_socket.ex

```diff
@@ -19,8 +21,17 @@ defmodule Yggdrasil.UserSocket do
   #
   # See `Phoenix.Token` documentation for examples in
   # performing token verification on connect.
-  def connect(_params, socket) do
-    {:ok, socket}
+  def connect(%{"token" => token}, socket) do
+    # user id is stored in the token
+    case Phoenix.Token.verify(socket, "user socket", token, max_age: 1209600) do
```

Comment:
Instead of commenting that the user id is in the token (which we can kind of infer from the case match clauses below, maybe comment on the length of the max_age.